content/academy/deploying_your_code/output_schema.md

```diff
@@ -0,0 +1,174 @@
+                            "___EXAMPLE_PICTURE_URL_FIELD___",
+                            "___EXAMPLE_LINK_URL_FIELD___",
+                            "___EXAMPLE_TEXT_FIELD___",
+                            "___EXAMPLE_BOOLEAN_FIELD___"
+                        ]
+                    },
+                    "display": {
+                        "component": "table",
+                        "properties": {
+                            "___EXAMPLE_NUMERIC_FIELD___": {
+                                "label": "ID",
+                                "format": "number"
+                            },
+                            "___EXAMPLE_PICTURE_URL_FIELD___": {
+                                "format": "image"
+                            },
+                            "___EXAMPLE_LINK_URL_FIELD___": {
+                                "label": "Clickable link",
+                                "format": "link"
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+To configure the output schema, simply replace the fields in the template with the relevant fields to you actor.
+
+For reference, you can use the [Zappos Scraper source code](https://github.com/PerVillalva/zappos-scraper-actor/blob/main/.actor/actor.json) as an example of how the final implementation of the output tab should look like in a live actor.
+
+```json
+{
+    "actorSpecification": 1,
+    "name": "zappos-scraper",
+    "title": "Zappos Scraper",
+    "description": "",
+    "version": "1.0.0",
+    "storages": {
+        "dataset": {
+            "actorSpecification": 1,
+            "title": "Zappos.com Dataset",
+            "description": "",
+            "views": {
+                "places": {
+                    "title": "Overview",
+                    "description": "It can take about one minute until the first results are available.",
+                    "transformation": {
+                        "fields": [
+                            "imgUrl",
+                            "brand",
+                            "name",
+                            "SKU",
+                            "inStock",
+                            "onSale",
+                            "price",
+                            "url"
+                        ]
+                    },
+                    "display": {
+                        "component": "table",
+                        "properties": {
+                            "imgUrl": {
+                                "label": "Product image",
+                                "format": "image"
+                            },
+                            "url": {
+                                "label": "Link",
+                                "format": "link"
+                            },
+                            "brand": {
+                                "label": "Brand",
+                                "format": "text"
```

Comment:
I'm not sure, but I think that you can skip the definitions if the only thing you do is capitalization of the label. It should be done automatically.

Reply:
I think it uses the variable name by default, but it wouldn't be capitalized.

Reply:
Sorry for delay. I keep forgetting to check GitHub now that I don't code often. It does capitalize. It even splits camel case into separate words and capitalizes all of them. See the difference between first number and second number. First has no entry in output schema, second is manually entered.

![CleanShot 2022-11-28 at 11 04 30@2x](https://user-images.githubusercontent.com/23726914/204250223-3d5ee6ef-0089-4819-8408-87b27e2b4365.png)

Reply:
Thanks, Ondra. I've updated my actor code and the output schema to reflect the correct usage of the labels. I also added a short note explaining the schema's default behavior.